sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/IdentityClientBase.java

```diff
@@ -500,6 +500,106 @@ AccessToken getTokenFromAzureCLIAuthentication(StringBuilder azCommand) {
         return token;
     }
 
+    AccessToken getTokenFromAzureDeveloperCLIAuthentication(StringBuilder azdCommand) {
```

Comment:
The logic in this method seems to have some common logic/flow as getTokenFromAzureCLIAuthentication method in this class. 
Good to refactor them to reuse same code.

Reply:
Yes, but we should not expect to keep a common behavior between the Azure cli and the Azure Developer CLI credentails.
I think it's better to keep their implementation apart as a hint about they can deviate at any point.